### PR TITLE
Adds serpent full core model

### PR DESCRIPTION
**Pull request by robfairh**
This PR addresses #5 

This PR adds the following files:
* EnergyDetector.png: spectrum figure
* fullcore: serpent input file of the full core model
* fullcore.png: figure with legends of the full core model
* fullcore_geom1.png: xy-plane of the geometry
* fullcore_geom2.png: yz-plane of the geomtry

This PR modifies the following files:
* README.md: changes description of the repo
* auxiliary.py: adds functions
* mhtgr.tex: adds table with characteristics of the full core model

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,4 +5,13 @@ The purpose of the model is to generate cross section for a Moltres input file o
 Also, the model in Serpent will be useful for comparing the results between Serpent and Moltres.
 
 This repository holds the following serpent input files:
-* compact: serpent model of the fuel compact.
\ No newline at end of file
+
+* compact: serpent model of the fuel compact.
+* fullcore: model of the full core. Reflector is defined as a block without the coolant channels.
```
Comment (ZoeRichter) [inline README.md:10]: Block of what, and is it rectangular or hexagonal?

File: README.md [hunk 2]
```diff
@@ -5,4 +5,13 @@ The purpose of the model is to generate cross section for a Moltres input file o
...
+* particles4.inp: defines the particle locations in the fuel compact.
+* particles79.inp: defines the particle locations in the fuel assembly.
```
Comment (ZoeRichter) [inline README.md:12]: What kind of particles? (I know these things, but for the README)

File: auxiliary.py [hunk 3]
```diff
@@ -45,9 +53,30 @@ def standard():
     plt.savefig("standard", dpi=300, bbox_inches="tight")
 
 
+def fullcore():
+    '''
+    Adds legend to fuel core geometry image:
+    'fullcore_geom1.png'.
+    '''
```
Comment (ZoeRichter) [inline auxiliary.py:60]: If you look through some of the other functions in the ARFC repos, there's a sort of template for this help section that might be helpful.  It's of the general form

```
    """ 
    Description of function
    Parameters:
    -----------
    var_a : type
        description of var_a
    Returns:
    --------
    var_b : type
        description of var_b
    """
```

**Review by ZoeRichter — APPROVED** [review]
I don't see any reason not to merge, I just had a few comments on the documentation.

**Review by mehmeturkmen — APPROVED** [review]

**Comment by mehmeturkmen** [comment]
This PR seems good. So no need to change anything.

**Pull request merged**